weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/schemaUtils.ts

```diff
@@ -639,7 +633,12 @@ export function filterRowsForNewDataset(
 
         const {[WEAVE_NAMESPACE]: weaveData, ...rest} = row;
         const filteredData = Object.fromEntries(
-          Object.entries(rest).filter(([key]) => targetFields.has(key))
+          Object.entries(rest).filter(([key]) =>
+            Array.from(targetFields).some(
+              schemaField =>
+                key === schemaField || key.startsWith(`${schemaField}.`)
+            )
+          )
```

Comment:
Include fields from the call if a parent field is included, eg included `inputs.example.row` if `inputs.example` was set